edx_proctoring/api.py

```diff
@@ -1044,7 +1044,16 @@ def update_attempt_status(exam_id, user_id, to_status,
     # call back to the backend to register the end of the exam, if necessary
     backend = get_backend_provider(exam)
     if backend:
-        # only proctored exams have a backend
+        # When onboarding exams change state to a completed status,
+        # look up any exams in the onboarding error states, and delete them.
+        # This will allow learners to return to the proctored exam and continue
+        # through the workflow.
+        if exam['is_practice_exam'] and backend.supports_onboarding and \
+                not ProctoredExamStudentAttemptStatus.is_incomplete_status(to_status):
```

Comment:
```suggestion
                ProctoredExamStudentAttemptStatus.is_complete_status(to_status):
```

Why the double-negative?

Reply:
I've removed the double negative. :)